packages/nimble-components/src/utilities/models/tracker.ts

```diff
@@ -0,0 +1,62 @@
+type ObjectFromList<T extends readonly string[], V = string> = {
```

Comment:
Providing the second type parameter (that defaults to `string` no less) seems unnecessary when the implementation relies on that type being a `boolean` (and is declared as such where it's used).